pkgs/tools/misc/odyssey/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "odyssey";
+  version = "2019-03-12";
```

```suggestion
  version = "unstable-2019-03-12";
```

[Nixpkgs manual package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming):
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

---

Fixed. I didn't know about this, thanks!